src/main/java/ca/bc/gov/backendstartapi/dto/UserDto.java

```diff
@@ -0,0 +1,55 @@
+package ca.bc.gov.backendstartapi.dto;
+
+import ca.bc.gov.backendstartapi.response.BaseResponse;
+import ca.bc.gov.backendstartapi.util.Empty;
+import java.util.Objects;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/** This class represents a User data transition object. */
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDto implements BaseResponse, Empty {
+
+  @Size(min = 2, max = 20)
+  @NotBlank
+  private String firstName;
+
+  @Size(min = 2, max = 20)
+  @NotBlank
+  private String lastName;
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    UserDto userDto = (UserDto) o;
+    return firstName.equals(userDto.firstName) && lastName.equals(userDto.lastName);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(firstName, lastName);
+  }
+
+  @Override
+  public String toString() {
+    String template = "UserDto{firstName='%s', lastName='%s'}";
```

Comment:
Yes. it can.

Reply:
But I guess we should consider having a `template` String instantiated all the time, in all instances, rather than only when calling `toString` method. Right? Since we're not going to use this template often, I think we can have it here, inside `toString`.